server/routes/utils/adHelpers.ts

```diff
@@ -325,15 +325,18 @@ export const appendTaskInfo = (
   return detectorMapWithTaskInfo;
 };
 
-// Four checks/transformations need to be made here:
+// Five checks/transformations need to be made here:
```

Comment:
minor: change `Five` to `Following`, so that we don't need to change it every time we have new checks.

Reply:
Agreed, changed.